draft-ietf-core-senml.md

```diff
@@ -115,7 +125,7 @@ informative:
     date: 2013 
     target: http://unitsofmeasure.org/ucum.html 
     seriesinfo:
-      Regenstrief Institute and Indiana University School of: Informatics 
+      Regenstrief Institute and Indiana University School of Informatics
```

Comment:
fix the space. When I look at the ref, it does not seem to be a series so not sure this is right and look like it might have changed to Regenstrief Institute, Inc. and the UCUM Organization. It also look like it has been dead for past few years. Decided to just ignore all that and fix the space